index.d.ts

```diff
@@ -0,0 +1,79 @@
+declare module "discord-webook-node" {
+  export type WebhookOptions = {
+    url: string;
+    throwErrors?: boolean;
+    retryOnLimit?: boolean;
+  };
+  export class Webhook {
+    constructor(options: string | WebhookOptions);
+    setUsername(username: string): void;
+    setAvatar(avatar: string): void;
+    sendFile(filePath: string): Promise<void>;
+    send(message: MessageBuilder): Promise<void>;
+    info(
+      title: string,
+      fieldName: string,
+      fieldValue: string,
+      inline: boolean
```

Comment:
"inline" is optional.
```suggestion
      inline?: boolean
```